app/forms/backtaxes_form.rb

```diff
@@ -1,7 +1,21 @@
 class BacktaxesForm < QuestionsForm
   set_attributes_for :intake, :needs_help_2016, :needs_help_2017, :needs_help_2018, :needs_help_2019
+  validate :at_least_one_year
 
   def save
     @intake.update(attributes_for(:intake))
   end
+
+  private
+
+  def at_least_one_year
+    chose_one = needs_help_2016 == "yes" ||
+      needs_help_2017 == "yes" ||
+      needs_help_2018 == "yes" ||
+      needs_help_2019 == "yes"
+    unless chose_one
+      errors.add(:needs_help_2016, "Please pick at least one year.")
+    end
+    chose_one
```

Comment:
FYI - no need to return false here for the validation to work. The object will be invalid if any errors are added to it.